eventservice/api/utils/group_auth.py

```diff
@@ -80,4 +82,24 @@ def check_permission_access_event(event, include_private_events, group_ids):
         # check public group
         if event and event.get('isGroupPrivate') is True:
             return False
-    return True
\ No newline at end of file
+    return True
+
+
+def check_all_group_event_admin():
+    is_all_group_event = False
+    if 'user_token' in g and not g.user_token_data.get('anonymous'):
+        if g.user_token_data.get('permissions'):
+            if g.user_token_data.get('permissions').lower() == ALL_GROUP_EVENTS:
```

Comment:
@shurwit may we know `g.user_token_data.get('permissions')` will return a list or a comma separated string?

Reply:
Stephen replied in Slack that this is comma-separated string. Please update the code accordingly. Thanks.

Reply:
updated